install.sh

```diff
@@ -104,12 +104,12 @@ fi
 rprefix=$(realpath -m "$root/$prefix")
 if ! $nonroot; then
     retc="$root/etc"
-    rsysconfdir="$root/$sysconfdir"
+    rsysconfdir=$(realpath -m "$root/$sysconfdir")
```

Comment:
This is a little dangerous. If /etc/sysconfig is a symlink on the build system (when building an rpm/deb), it will translate that symlink on the build host, even though the translation may be incorrect for the installed system.

Reply:
You are right. I will remove this change